client/a8c-for-agencies/sections/referrals/referral-details/mobile/style.scss

```diff
@@ -0,0 +1,41 @@
+.referral-purchases-mobile {
+	border: 1px solid var(--color-neutral-10);
+	border-radius: 4px;
+	margin-block-end: 1rem;
+	padding: 1rem;
+	display: flex;
+	flex-direction: column;
+	gap: 1rem;
+
+	.referral-purchases-mobile__header {
+		display: flex;
+		justify-content: space-between;
+
+		.gridicon {
+			transform: rotate(90deg);
+		}
+	}
+
+	.referral-purchases-mobile__product-name {
+		font-weight: 500;
+		font-size: 0.875rem;
+		color: var(--color-gray-80);
+	}
+
+	h3 {
+		font-weight: 500;
+		font-size: 11px; // stylelint-disable-line declaration-property-unit-allowed-list
```

Comment:
Non-blocker:

I think we can do `font-size: rem(11px);` instead

Reply:
Oh, nice, I haven't known about that! Thanks!